base/range.jl

```diff
@@ -182,33 +183,33 @@ end
 
 function linspace{T<:AbstractFloat}(start::T, stop::T, len::T)
```

Comment:
Out of curiosity (@StefanKarpinski), why isn't it `::Integer`? Might this just be a copy/paste issue?